CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -47,16 +58,21 @@ public IList<HarvestDeveloper> RetrieveDevelopers()
             
             var apiResponse = response.Result;
             var users = apiResponse["users"];
-           
-            return users.Where(user => (bool)user["is_active"] != false)
-                .Select(developer => new HarvestDeveloper()
-                    {
-                        Id = (int) developer["id"],
-                        FirstName = developer["first_name"].ToString(),
-                        LastName = developer["last_name"].ToString(),
-                        Email = developer["email"].ToString()
-                    }
-                ).ToList();
+
+            var activeDevelopers = users.Where(user => (bool)user["is_active"] && IsDeveloper(user));
+            return activeDevelopers.Select(developer => new HarvestDeveloper
+                {
+                    Id = (int) developer["id"],
+                    FirstName = developer["first_name"].ToString(),
+                    LastName = developer["last_name"].ToString(),
+                    Email = developer["email"].ToString()
+                }
+            ).ToList();
+        }
+        
+        private bool IsDeveloper(JToken user)
+        {
```

Comment:
What will happen here if there's no roles array? Is that a possibility?

Reply:
I believe Harvest has an empty `roles` array if a person has no roles.

Reply:
OK - good to know.